java-samples/corda5-negotiation-cordapp/workflows/build.gradle

```diff
@@ -12,6 +12,9 @@ plugins {
 // A cordaProvided declaration is required for anything that we use that the Corda API provides.
 // This is required to allow us to build CorDapp modules as OSGi bundles that CPI and CPB files are built on.
 dependencies {
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.8.1'
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.8.1'
```

Comment:
want to add the version values to the gradle.properties?

Reply:
Actually, this was not needed as below there is a code line that gets it from the gradle.properties